src/karma-webpack.js

```diff
@@ -210,11 +211,12 @@ function Plugin(
   compiler.hooks.done.tap(this.plugin, done.bind(this));
   compiler.hooks.invalid.tap(this.plugin, invalid.bind(this));
 
-  webpackMiddlewareOptions.publicPath = path.join(
-    '/',
-    '_karma_webpack_',
-    '/'
-  );
+  webpackMiddlewareOptions.publicPath = path.join('/', '_karma_webpack_', '/');
+
+  // Set webpack's color config to value specified in Karma's config for consistency
+  webpackMiddlewareOptions.stats = webpackMiddlewareOptions.stats || {};
+  webpackMiddlewareOptions.stats.colors = colors;
```

Comment:
Will this always override the default `webpack` colors config ? This would be a breaking change then... Could this eventually be made an option intead ? e.g

```js
if (config.colors) {
  webpackDevMiddlewareOptions.stats.colors = colors
}
```

> ⚠️ Not sure if this works correctly, but to illustrate the idea :)

Reply:
@michael-ciniawsky What do you mean by a breaking change?

It should work in a similar way that webpack's watch config is being overridden by Karma's singleRun config https://github.com/webpack-contrib/karma-webpack/blob/master/src/karma-webpack.js#L56

If you specify `colors: false` on Karma, you probably don't want colors on webpack's output either

Reply:
Below are some images to illustrate the changes

I have a simple project setup you can play around with https://github.com/spicalous/color-example

First image: No configs (Karma and webpack output color)
![default-no-color-opt](https://user-images.githubusercontent.com/4274552/45585619-59e1fb00-b8df-11e8-914f-318367c83409.png)

Second image: Karma color:false (Karma outputs no color, webpack outpus color)
![karma-color-false](https://user-images.githubusercontent.com/4274552/45585625-6cf4cb00-b8df-11e8-8425-13f95ce4ca11.png)

Third image: Karma color:false with these changes (Karma and webpack both output no color)
![karma-color-false-after](https://user-images.githubusercontent.com/4274552/45585630-839b2200-b8df-11e8-98b2-a93e586ff651.png)

Reply:
`webpackMiddlewareOptions.stats` could be `string`. This breaks configuration i.e.:
```js
  //...
  stats: 'errors-only'
```

Reply:
Fixed in `rc.5`